src/js/virtual-list.js

```diff
@@ -18,7 +18,7 @@ var VirtualList = function (listBlock, params) {
 
     // Preparation
     var vl = this;
-    vl.listBlock = $(listBlock);
+    vl.listBlock = typeof listBlock=="string"?$(listBlock):listBlock;
     vl.params = params;
```

Comment:
What is this doing and why?

You mentioned DOM7 but this isn't part of the DOM7 library